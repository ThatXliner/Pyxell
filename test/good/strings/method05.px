
s = "qwerty"
print s.get(0), s.get(5)!
print s.get(-4)?.code
print s.get(6), s.get(-7)?.code
