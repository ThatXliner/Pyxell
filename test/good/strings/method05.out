'q' y
101
null null
